admin/model_search_index_response.go

```diff
@@ -0,0 +1,494 @@
+// Code based on the AtlasAPI V2 OpenAPI file
+
+package admin
+
+import (
+	"encoding/json"
+)
+
+// SearchIndexResponse struct for SearchIndexResponse
+type SearchIndexResponse struct {
+	// Label that identifies the collection that contains one or more Atlas Search indexes.
+	CollectionName *string `json:"collectionName,omitempty"`
+	// Label that identifies the database that contains the collection with one or more Atlas Search indexes.
+	Database *string `json:"database,omitempty"`
+	// Unique 24-hexadecimal digit string that identifies this Atlas Search index.
+	IndexID                 *string                       `json:"indexID,omitempty"`
+	LatestDefinition        *VectorSearchIndexDefinition  `json:"latestDefinition,omitempty"`
```

Comment:
`LatestDefinition` is fixed to VectorSearchIndexDefinition, which is missing properties associated to the TextSearchIndexDefinition view.

Reply:
I have fixed that by transformation in #342